### PR TITLE
Bump eventing kafka broker manifests to 1.3

### PR DESCRIPTION
**Pull request by pierDipi**
As per title.

See individual commits for details.

### CODE REVIEW

File: knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml [hunk 1]
```diff
@@ -767,6 +834,66 @@ data:
       </root>
     </configuration>
 ---
+# Copyright 2019 The Knative Authors
+#
```
Comment (pierDipi) [inline knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml:838]: Note: we should remove this from midstream 1.3

Comment (matzew) [reply knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml:838]: _this_ ?

Comment (matzew) [reply knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml:838]: yeah, we used to remove that via patch

**Comment by pierDipi** [comment]
Fix https://github.com/openshift-knative/eventing-kafka-broker/pull/311

File: knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml [hunk 2]
```diff
@@ -229,7 +229,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   labels:
-    kafka.eventing.knative.dev/release: "v20211214-f1bb19a50"
+    kafka.eventing.knative.dev/release: "v20220302-f4dfb1493"
```
Comment (matzew) [inline knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml:232]: Ha! this is when you redistribute downloaded files :smile:

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Comment by pierDipi** [comment]
infra 

/test 4.10-upgrade-tests-aws-ocp-410

**Comment by pierDipi** [comment]
infra

/test 4.10-upgrade-tests-aws-ocp-410

**Pull request merged**